src/pages/en/guides/markdown-content.md

```diff
@@ -35,11 +39,14 @@ To learn more about adding a layout to your page, read the next section on **Mar
 
 ### Frontmatter `layout`
 
-In Astro, Markdown and MDX pages have a special frontmatter property for `layout` that defines the relative path to an Astro [layout component](/en/core-concepts/layouts/). This component will wrap your Markdown content, providing a page shell and any other included page template elements.
+In Astro, Markdown and MDX pages have a special frontmatter property for `layout` that defines the relative path to an Astro [layout component](/en/core-concepts/layouts/#markdown-layouts). This component will wrap your Markdown content, providing a page shell and any other included page template elements.
 
-```markdown
+```markdown {6} 
 ---
 // src/pages/page.md
+title: "Astro v1 Launch!"
+author: "Matthew Phillips"
+date: "09 Aug 2022"
 layout: ../layouts/BaseLayout.astro
```

Comment:
I'm not sure that it matters, since it's highlighted... but I have no objections! I think a lot of legacy projects will have it at the top anyway.  ;)